v3-docs/v3-migration-docs/index.md

```diff
@@ -15,15 +15,15 @@ Meteor 3.0 is currently in its Release Candidate (RC) phase, a nearly final vers
 **Node.js version:** `20.11.1 LTS` <br/>
 **NPM version:** `10.2.4`
 
-## How to test Meteor 3.0?
+## How to try Meteor 3.0?
 
 You can create a new Meteor 3.0 project by running the command below:
 
 ```bash
 meteor create my-new-project --release 3.0-rc.1
 ```
 
-## How to update from version 2?
+## How to upgrade from Meteor 2.x?
```

Comment:
Also, `upgrade` is more accurate since it's a major version

Reply:
It is a little confusing since the command says `update`, but using the correct wording where we can could be better for SEO, etc.